specification/maps/data-plane/Search/preview/2022-09-01-preview/examples/ReverseGeocoding.json

```diff
@@ -0,0 +1,86 @@
+{
+  "parameters": {
+    "api-version": "2022-09-01-preview",
+    "query": "-122.138681,47.630358",
+    "AzureKey": "[subscription-key]"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "geoJsonData": {
+          "type": "FeatureCollection",
+          "features": [
+            {
+              "type": "Feature",
+              "properties": {
+                "address": {
+                  "countryRegion": {
+                    "name": "United States"
+                  },
+                  "adminDistricts": [
+                    {
+                      "shortName": "WA"
+                    },
+                    {
+                      "shortName": "King County"
+                    }
+                  ],
+                  "formattedAddress": "15127 NE 24th St, Redmond, WA 98052",
+                  "locality": "Redmond",
+                  "postalCode": "98052",
+                  "addressLine": "15127 NE 24th St"
+                },
+                "type": "Address",
+                "confidence": "Medium",
+                "matchCodes": [
+                  "Good"
+                ],
+                "geocodePoints": [
+                  {
+                    "geometry": {
+                      "type": "Point",
+                      "coordinates": [
+                        -122.138681,
+                        47.630358
+                      ]
+                    },
+                    "calculationMethod": "Rooftop",
+                    "usageTypes": [
+                      "Display"
+                    ]
+                  },
+                  {
+                    "geometry": {
+                      "type": "Point",
+                      "coordinates": [
+                        -122.1386787,
+                        47.6302179
+                      ]
+                    },
+                    "calculationMethod": "Rooftop",
+                    "usageTypes": [
+                      "Route"
+                    ]
+                  }
+                ]
+              },
+              "geometry": {
+                "type": "Point",
+                "coordinates": [
+                  -122.138681,
+                  47.630358
+                ]
+              },
+              "bbox": [
+                -122.14632282407,
+                47.626495282429325,
+                -122.13103917593001,
+                47.63422071757068
+              ]
+            }
+          ]
+        }
+      }
+    }
+  }
```

Comment:
At the end, do we want to show some trace id for debugging purposes?

Reply:
I have a discussion with Chun-Ting Wu (Nick) about this. 
I think there is no need for us to dive into the log in RevGC.
Further, if we got failed requests, we can get the external url from our log, and try with them.  
Therefore, I think we don't need to have traceId in Azure maps service.
Do you have any idea or suggestion for this?